examples/all-clusters-minimal-app/nrfconnect/main/AppTask.cpp

```diff
@@ -74,6 +80,10 @@ bool sHaveBLEConnections   = false;
 #ifdef CONFIG_CHIP_CRYPTO_PSA
 chip::Crypto::PSAOperationalKeystore sPSAOperationalKeystore{};
 #endif
+
+#ifdef CONFIG_NET_L2_OPENTHREAD
+app::Clusters::NetworkDriverObj<DeviceLayer::NetworkCommissioning::GenericThreadDriver> threadNetworkDriver(0 /*endpointId*/);
```

Comment:
This is minor, but if you could change the name here and in other places btw of some other changes.
```suggestion
app::Clusters::NetworkDriverObj<DeviceLayer::NetworkCommissioning::GenericThreadDriver> sThreadNetworkDriver(0 /*endpointId*/);
```